infra/ansible/inventories/prod/group_vars/all.yml

```
@@ -0,0 +1,3 @@
+ansible_user: chanyong
```

아래와 같이 하거나, 
```suggestion
ansible_user: $USER
```
아니면, 일반적인 개인정보를 포함하지 않도록 합니다.
```suggestion
ansible_user: user
```